en/config/safety.md

```diff
@@ -227,6 +229,49 @@ Parameter | Description
 [VT_FW_ALT_ERR](../advanced_config/parameter_reference.md#VT_FW_ALT_ERR) | Maximum negative altitude error for fixed wing flight. If the altitude drops more than this value below the altitude setpoint the vehicle will transition back to MC mode and enter failsafe RTL.
 
 
+## Safety Switches {#safety_switch}
+
+A safety switch allows you to immediately stop all motors or return the vehicle from the remote control transmitter
+(if you lose control of the vehicle, this may be better than allowing it to continue flying).
+
+This section lists the available safety switches.
+
+### Kill Switch {#kill_switch}
+
+A kill switch immediately stops all motor outputs (if flying the vehicle will start to fall).
+The vehicle is not disarmed, and the motors will restart if the switch is reverted.
+
+The kill switch is enabled as part of *QGroundControl* [Flight Mode Setup](../config/flight_mode.md#safety_switch).
+
+
+### Arm/Disarm Switch {#arming_switch}
+
+The arm/disarm switch is a replacement for the default stick arming/disarming mechanism (and serves the same purpose: making sure there is an intentional step involved before the motors start/stop). It might be used in preference to the default mechanism because:
+- Of a preference of a switch over a stick motion (e.g. if using a stick on another autopilot).
+- It avoids accidentally triggering arming/disarming in-air with a certain stick motion.
+- There is no delay (it reacts immediately).
+
+The arm/disarm switch immediately disarms (stop) motors for those [flight modes](../getting_started/flight_modes.md) that *support disarming in flight*. This includes: 
+- *Manual mode*
+- *Acro mode*
+- *Stabilized*
+- *Rattitude*
+
+For modes that do not support disarming in flight, the switch is ignored during flight, but may be used after landing is detected. This includes *Position mode*.
```

Comment:
We might want to mention autonomous modes (such as mission) as well?

Reply:
Yes. I'll just change to "This includes *Position mode* and autonomous modes (e.g. *Mission*, *Land* etc.)". Thanks for review!